docs/source/contributing/contributing-guide.rst

```diff
@@ -0,0 +1,55 @@
+************************
+Contributing Guide
+************************
+
+Thanks for checking out the contributing guide. Substra warmly welcomes contributions!
+
+Ground rules & expectations
+===========================
+
+* Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on how things should be done. Try to listen to others rather than convince them that your way is correct.
+* Substra has a :doc:`Contributor Code of Conduct </contributing/code-of-conduct>`. By participating in this project, you agree to abide by its terms.
+
+How to contribute
+=================
+
+You should usually open a pull request in the following situations:
+
+* Submit trivial fixes (for example, a typo, a broken link or an obvious error)
+* Start work on a contribution that was already asked for, or that you've already discussed, in an issue
+
+A pull request doesn't have to represent finished work. You can open a pull request early on, so others can watch or give feedback on your progress. Just open it as a "draft". You can always add more commits later.
+
+Here's how to submit a pull request:
+
+* `Fork the repository <https://guides.github.com/activities/forking/>`_ and clone it locally. Connect your local to the original "upstream" repository by adding it as a remote. Pull in changes from "upstream" often so that you stay up to date so that when you submit your pull request, merge conflicts will be less likely. (See more detailed instructions `here <https://help.github.com/articles/syncing-a-fork/>`_.)
+* `Create a branch <https://guides.github.com/introduction/flow/>`_ for your edits.
+* **Sign off** your commits.
+* **Test your changes.** Please ensure that your contribution passes all tests if you open a pull request. If there are test failures, you will need to address them before we can merge your contribution.
+* **Contribute in the style of the project** to the best of your abilities. This may mean using indents, semi-colons or comments differently than you would in your own repository, but makes it easier for us to merge, others to understand and maintain in the future.
+
+Sign Off
```

Comment:
@Milouu we should add something about the contributors file. Every contributor should be mentioned in a CONTRIBUTORS.MD file and then in the LICENCE we mention the substra contributors [here](https://github.com/Substra/substra-documentation/blob/main/LICENSE#L179) instead of Owkin Inc.